<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UBControl</class>
 <widget class="QWidget" name="UBControl">
  <property name="modal" stdset="0">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>250</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="simulationBox">
       <property name="title">
        <string>Network Simulation</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="simpleButton">
            <property name="text">
             <string>&amp;Simple</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="Line" name="line_1">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="rangeLabel">
            <property name="text">
             <string>Range (in meter):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rangeEdit">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="ns3Button">
            <property name="text">
             <string>&amp;ns-3</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="networkBox">
       <property name="title">
        <string>Agent Network</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="numberLabel">
            <property name="text">
             <string>Agent Number:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="numberEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="disconnectButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Disconnect</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="connectButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Connect</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>numberEdit</tabstop>
  <tabstop>connectButton</tabstop>
  <tabstop>disconnectButton</tabstop>
  <tabstop>ns3Button</tabstop>
  <tabstop>simpleButton</tabstop>
  <tabstop>rangeEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
